### PR TITLE
fix: update goal styling

### PR DESCRIPTION
**Pull request by ciduarte**
edX outline tab with goals feature

AA-1090

### CODE REVIEW

File: src/course-home/outline-tab/widgets/WeeklyLearningGoalCard.jsx [hunk 1]
```diff
@@ -47,68 +47,62 @@ function WeeklyLearningGoalCard({
   }
 
   return (
-    <div className="row w-100 m-0 p-0">
```
Comment (ciduarte) [inline src/course-home/outline-tab/widgets/WeeklyLearningGoalCard.jsx:50]: Removed this `<div>` and pulled the `row` into the Card. Let me know if you want me to call out all the changes in this file

File: src/course-home/outline-tab/widgets/WeeklyLearningGoalCard.jsx [hunk 2]
```diff
@@ -47,68 +47,62 @@ function WeeklyLearningGoalCard({
...
-        className="mb-3 shadow border-0"
-        data-testid="weekly-learning-goal-card"
-      >
-        <Card.Body className="p-3.5">
-          <Card.Title>
-            <h2 className="h4 m-0 text-primary-500">{intl.formatMessage(messages.setWeeklyGoal)}</h2>
-          </Card.Title>
-          <Card.Text
-            className="text-gray-700"
-              // id="learning_goal_choices_label"
```
Comment (ciduarte) [inline src/course-home/outline-tab/widgets/WeeklyLearningGoalCard.jsx:61]: removed this lingering comment

File: src/course-home/outline-tab/widgets/WeeklyLearningGoalCard.jsx [hunk 3]
```diff
@@ -47,68 +47,62 @@ function WeeklyLearningGoalCard({
   }
 
   return (
-    <div className="row w-100 m-0 p-0">
-      <Card
-        className="mb-3 shadow border-0"
-        data-testid="weekly-learning-goal-card"
-      >
-        <Card.Body className="p-3.5">
```
Comment (ciduarte) [inline src/course-home/outline-tab/widgets/WeeklyLearningGoalCard.jsx:55]: updated this padding for responsiveness

File: src/course-home/outline-tab/widgets/WeeklyLearningGoalCard.jsx [hunk 4]
```diff
@@ -47,68 +47,62 @@ function WeeklyLearningGoalCard({
...
-              </div>
-              <div className="col">
-                {intl.formatMessage(messages.goalReminderDetail)}
-              </div>
+            <small>{intl.formatMessage(messages.setGoalReminder)}</small>
+          </Form.Switch>
+        </div>
+      </Card.Body>
+      {isGetReminderSelected && (
+        <Card.Footer className="border-0 px-2.5 bg-light-200 ">
```
Comment (MatthewPiatetsky) [inline src/course-home/outline-tab/widgets/WeeklyLearningGoalCard.jsx]: nit extra space?

**Review by MatthewPiatetsky — APPROVED** [review]
A bit hard to tell for me just from the screenshot what was fixed but I trust you and nothing looks very off - if you could verify in prod that it was fixed after merged looks good!

**Pull request merged**